ethereum/oracles/src/noir/circuit/verifierData.ts

```diff
@@ -0,0 +1,29 @@
+import { Abi, InputMap, abiEncode } from '@noir-lang/noirc_abi';
+import { readTomlObject } from '../../util/file.js';
+import { Hex } from 'viem';
+import { CircuitAbi } from './abi.js';
+
+export class VerifierData {
+  public static async create(verifierTomlPath: string, abi: Abi) {
```

Comment:
`fromFile`

Reply:
But it's not from file, it's from file & abi.
create pattern is like constructor, but async. I think it makes sense to add names constructors if we have many to distinguish, but here we have just one